loader/loader-for-all-content/loader-for-all-content/Pages/Problem.razor

```diff
@@ -0,0 +1,17 @@
+﻿@page "/problem"
```

Comment:
It is questionable, whether the default behavior should be to forbid the scroll. So, if we could use other naming than problem, will be perfect:)

Reply:
Will do